### ISSUE TITLE
Automating FAA Chart Download and Sorting by Airport and Procedure

### ISSUE THREAD

**Issue by jesussmile** [JanC/aeronav-dtpp#4]
Hello there,

My objective is to download FAA charts and sort them based on the airport and its procedures. I am currently trying out your project. So far, I have only been able to retrieve the XML file.

Regarding the sorting of georeferenced PDFs, is this handled by the project, or do I need to download and sort them manually? Alternatively, is the project primarily just an XML parser?

I am sure it may have changed a bit on how this project was initially intended to run ?
I am looking for a way to rename FAA charts based on their corresponding airport and procedure names.

```
(base) PS C:\Users\pannam> anm-dtpp fetch
INFO: Downloading https://nfdc.faa.gov/webContent/dtpp/current.xml
INFO: Writing xml to ./output/current.xml
INFO: Done
(base) PS C:\Users\pannam> anm-dtpp charts --icao KSBP
Usage: anm-dtpp charts [OPTIONS]
Try 'anm-dtpp charts --help' for help.

Error: Invalid value for '--db-file' / '-d': File './output/dtpp.sqlite' does not exist.
(base) PS C:\Users\pannam> 
```

**Comment by JanC** [comment]
hey there,
it's been very long time I haven't used this project but from what I see, you need to run the `anm-dtpp create` command first. This will read the downloaded xml file and create a database needed for the `anm-dtpp charts --icao KSBP` command

**Comment by jesussmile** [comment]
Yes, sir! Got it, it works fine now. In fact, it's genius how you got it to work. I want to move forward with the project and try creating georeferenced plates for airport charts as well.

Could you give me a basic idea of where you last stopped? From your previous conversation, you mentioned using Inkscape to get the location of text and using regex to extract it. Could you provide a rough idea of how you used Inkscape to do this? I've never used it before, so just a small bit of guidance in the right direction would be very helpful.

**Comment by JanC** [comment]
> From your previous conversation, you mentioned using Inkscape to get the location of text and using regex to extract it
What conversation are you reffering to?

There this project https://github.com/jlmcgraw/GeoReferencePlates that is creating georeferenced plates but I never really could make it work. It also seems the maintainer is not maintaining the project anymore

I didn't plan to do anything relted to georeferencing
